website/pages/intro/index.mdx

```diff
@@ -32,7 +32,7 @@ Nomad is widely adopted and used in production by PagerDuty, Target, Citadel, Tr
 
 - **Deploy Containers and Legacy Applications**: Nomad’s flexibility as an orchestrator enables an organization to run containers, legacy, and batch applications together on the same infrastructure. Nomad brings core orchestration benefits to legacy applications without needing to containerize via pluggable [task drivers](/docs/drivers).
 
-- **Simple & Reliable**: Nomad runs as a single 75MB binary and is entirely self contained - combining resource management and scheduling into a single system. Nomad does not require any external services for storage or coordination. Nomad automatically handles application, node, and driver failures. Nomad is distributed and resilient, using leader election and state replication to provide high availability in the event of failures.
+- **Simple & Reliable**: Nomad runs as a single 120MB binary and is entirely self contained - combining resource management and scheduling into a single system. Nomad does not require any external services for storage or coordination. Nomad automatically handles application, node, and driver failures. Nomad is distributed and resilient, using leader election and state replication to provide high availability in the event of failures.
```

Comment:
do size really matter? 

```suggestion
- **Simple & Reliable**: Nomad runs as a small binary and is entirely self contained - combining resource management and scheduling into a single system. Nomad does not require any external services for storage or coordination. Nomad automatically handles application, node, and driver failures. Nomad is distributed and resilient, using leader election and state replication to provide high availability in the event of failures.
```

Reply:
I'm inclined to agree with @jippi here, if for no other reason than we don't have to keep updating it because the binary size has changed over time.

Reply:
:+1: to removing the size as we're likely to keep forgetting to update it (I still dream of shipping a no-builtin-plugins version someday too that would complicate this story :sweat_smile: )

```suggestion
- **Simple & Reliable**: Nomad runs as a single binary and is entirely self contained - combining resource management and scheduling into a single system. Nomad does not require any external services for storage or coordination. Nomad automatically handles application, node, and driver failures. Nomad is distributed and resilient, using leader election and state replication to provide high availability in the event of failures.
```